src/sentry/api/endpoints/debug_files.py

```diff
@@ -370,6 +373,29 @@ def get(self, request: Request, project) -> Response:
         return Response({"missing": missing})
 
 
+@region_silo_endpoint
+class DebugFilesConfigEndpoint(ProjectEndpoint):
+    owner = ApiOwner.HYBRID_CLOUD
+    publish_status = {"GET": ApiPublishStatus.PRIVATE}
+    permission_classes = (ProjectReleasePermission,)
+
+    def get(self, request: Request, project) -> Response:
```

Comment:
Is the plan to update sentry-cli to use this endpoint to locate the upload URL?

Reply:
Yep, I'm currently testing the change that consumes this new API in the CLI and uses this for legacy uploads as well. What sucks is that existing CLI users will have to update to fix timeouts/broken uploads. 

As an alternative approach, is there a way we could configure the API proxy to issue a redirect to the correct region vs simply proxying the request?

Reply:
We might be able to do a redirect. I suspect we have made clients that won't respect redirects though.